libyul/backends/evm/AbstractAssembly.h

```diff
@@ -110,6 +110,9 @@ class AbstractAssembly
 	/// Appends an assignment to an immutable variable.
 	virtual void appendImmutableAssignment(std::string const& _identifier) = 0;
 
+	/// Appends data to the very end of the bytecode. Repeated calls concatenate.
+	virtual void appendAuxiliaryDataToEnd(bytes const& _data) = 0;
```

Comment:
"append" already implies that it's at the end so "to end" looks superfluous. Maybe that's good because it does make you think there's something unusual going on but still, the name sounds weird to me.

I guess the difference you want to underscore here is that this data will always end up at the end even if other `appendXXX()` functions get called later? In that case I think we could call it "trailing data". To me it implies that you not only append it at the end but also that it always stays there:
```suggestion
	virtual void appendTrailingData(bytes const& _data) = 0;
```

Reply:
The "auxiliary" means that it is not regular data. I'll rename it to "appendToAuxiliaryData".